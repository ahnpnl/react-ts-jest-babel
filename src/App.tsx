import React from 'react';
import './App.css';

const App: React.FC = () => {
  return (
    <>Test</>
  );
};

export default App;
