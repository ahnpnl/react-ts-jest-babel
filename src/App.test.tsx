import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import styles from './custom-style.scss';

test('renders learn react link', () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/learn react/i);
  expect(styles).toBeDefined();
  expect(linkElement).toBeInTheDocument();
});
