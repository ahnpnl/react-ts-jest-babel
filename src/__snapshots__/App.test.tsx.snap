// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title component 1`] = `
Array [
  <h1
    className="mainTitle"
  >
    <strong>
      Jest Example
    </strong>
  </h1>,
  <h3
    className="subTitle"
  >
    React with CSS Modules automated tests
  </h3>,
]
`;
